lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -86,8 +86,6 @@
 #define TEE_STORAGE_PRIVATE_REE	 0x80000000
 /* Storage is the Replay Protected Memory Block partition of an eMMC device */
 #define TEE_STORAGE_PRIVATE_RPMB 0x80000100
-/* Storage is provided by a SQLite database in the normal world filesystem */
-#define TEE_STORAGE_PRIVATE_SQL  0x80000200
```

Comment:
Reserving the value makes sense, but using a pragma to emit a warning instead of compile error seems a bit over engineered. Lets assume a TA is using this define, when the TA is compiled there will be a warning which is turned into an error since the TA is compiled with -Werror.

How about:
```
/* Was TEE_STORAGE_PRIVATE_SQL, which isn't supported any longer */
#define TEE_STORAGE_PRIVATE_SQL_RESERVED  0x80000200
```